app/controllers/doorkeeper/openid_connect/discovery_controller.rb

```diff
@@ -89,7 +89,7 @@ def keys_response
 
         {
           keys: [
-            signing_key.slice(:kty, :kid, :e, :n).merge(
+            signing_key.slice(:kty, :kid, :e, :n, :x, :y, :crv).merge(
```

Comment:
They are defined in the OpenID Connect / JWKS specs, which also define the endpoint. They aren't short var names, they are the x and the y of the Elliptic Curve.